firecracker/microvm.py

```diff
@@ -262,6 +267,34 @@ def enable_rootfs(self, path_on_host: str) -> str:
             rootfs_filename = Path(path_on_host).name
             jailer_path_on_host = f"/opt/{rootfs_filename}"
             os.link(path_on_host, f"{self.jailer_path}/{jailer_path_on_host}")
+            return Path(jailer_path_on_host)
+        else:
+            return path_on_host
+
+    def mount_rootfs(self, path_on_host: Path) -> Path:
+        """Mount a rootfs to the VM.
+        """
+        self.mounted_fs = path_on_host
+        if self.use_jailer:
+            rootfs_filename = Path(path_on_host).name
+            jailer_path_on_host = Path(f"{DEVICE_BASE_DIRECTORY}/{rootfs_filename}")
```

Comment:
Nitpick: instead of using an f-string, you can do the following:


```suggestion
            jailer_path_on_host = Path(DEVICE_BASE_DIRECTORY) / rootfs_filename
```